### DISCUSSION TITLE
Fulgurium Stun Darts can't be stored in the bandolier.

### DISCUSSION THREAD

**Discussion by UlyssesBSD** [FakeFishGames/Barotrauma#14531] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Exactly what the title says. Regular darts can be stored just fine but not the fulgurium ones.

### Reproduction steps

1. Get bandolier
2. Get fulgurium darts
3. Try to store said darts in the bandolier

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/a0e4ae85a01f4100d6fcc100f898f2a813ac014d (commit in our private repo). Fix coming up in the next update.
